src/components/HeaderDisplay/index.tsx

```diff
@@ -202,6 +204,20 @@ function VersionSelector({ state }: VersionSelectorProps): React.ReactElement {
             },
           });
 
+          // Add the duplicate button
+          actions.push({
+            type: 'button',
+            icon: faCopy,
+            onClick: () => {
+              state.cloneVersion(
+                version.id,
+                getNextVersionName(
+                  state.allVersionNames.map(({ name }) => name)
+                )
```

Comment:
Thoughts on using this instead? I think it makes it clearer what the new schedule is, though feel free to disagree.

```suggestion
                `Copy of ${version.name}`
```

Reply:
Sounds good